src/pages/MainPage.jsx

```diff
@@ -1,14 +1,93 @@
+import Card from '@mui/material/Card';
```

Comment:
I don't where this MainPage is being imported. It looks like the app is running the same without it. If it's superfluous, it should be removed. The same thing with the jpeg being added to the project.

Reply:
I removed the extra files